certif/tests/unit/services/session_test.js

```diff
@@ -0,0 +1,197 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import sinon from 'sinon';
+
+module('Unit | Service | session', function (hooks) {
+  setupTest(hooks);
+
+  let sessionService;
+  let routerService;
+
+  hooks.beforeEach(function () {
+    sessionService = this.owner.lookup('service:session');
+    sessionService.currentDomain = { getExtension: sinon.stub() };
+    sessionService.currentUser = { load: sinon.stub(), certificationPointOfContact: null };
+
+    routerService = this.owner.lookup('service:router');
+    routerService.transitionTo = sinon.stub();
+  });
+
+  module('#handleAuthentication', function () {
+    test('calls loadCurrentUserAndSetLocale with no arg', async function (assert) {
+      // given
+      sessionService.loadCurrentUserAndSetLocale = sinon.stub();
+      sessionService.currentUser = { certificationPointOfContact: { isMemberOfACertificationCenter: false } };
+      sessionService._setLocale = sinon.stub();
+
+      // when
+      await sessionService.handleAuthentication();
+
+      // then
+      sinon.assert.calledWith(sessionService.loadCurrentUserAndSetLocale);
+      assert.ok(true);
+    });
+  });
+
+  module('#loadCurrentUserAndSetLocale', function () {
+    module('when domain is .fr', function () {
+      module('when no lang query parameter', function () {
+        module('when user is not loaded', function () {
```

Comment:
Il manque je pense le teste du `this.locale.hasLocaleCookie`

Reply:
Ce test a été ajouté dans le dernier push. Merci.